src/Compilers/Server/VBCSCompilerTests/CompilerServerTests.cs

```diff
@@ -188,136 +221,133 @@ private static ProcessResult RunCompilerOutput(TempFile file)
             return ProcessUtilities.Run(file.Path, "", Path.GetDirectoryName(file.Path));
         }
 
-        private static void VerifyResult(ProcessResult result)
+        private static void VerifyResult((int exitCode, string output) result)
         {
-            Assert.Equal("", result.Output);
-            Assert.Equal("", result.Errors);
-            Assert.Equal(0, result.ExitCode);
+            Assert.Equal("", result.output);
+            Assert.Equal(0, result.exitCode);
         }
 
-        private void VerifyResultAndOutput(ProcessResult result, TempDirectory path, string expectedOutput)
+        private void VerifyResultAndOutput((int exitCode, string output) result, TempDirectory path, string expectedOutput)
         {
             using (var resultFile = GetResultFile(path, "hello.exe"))
             {
                 VerifyResult(result);
 
+#if NET46
                 var runningResult = RunCompilerOutput(resultFile);
                 Assert.Equal(expectedOutput, runningResult.Output);
+#endif
```

Comment:
Consider adding `expectedOutput` as an argument to `RunCompilerOutput` and move `#if NET46` to that method.